Петров Евгений

LeMarck opened the pull request.

LeMarck committed:
update

LeMarck committed:
update

LeMarck committed:
update

Я всё проспал((

LeMarck committed:
codestyle

bratva commented on index.css:
```diff
@@ -1 +1,59 @@
+.elements,
+.cauldron
+{
+    display: flex;
```
Кажется, что flex нужен не всем этим элементам.

bratva commented:
@LeMarck Молодец! Мне нравится. 

Хочется сделать еще круче (это не обязательно, но будет круто если сделаешь)
1. Запретить выделение элементов в котле и списке (см [user-select](http://caniuse.com/#search=user-select))
2. Покрасить каждый элемент в свой стиль (вода - голубая, огонь - красный и тп)
3. Оформить котел и полку с элементами
4. Хочется какой-то анимации, когда получили результат

Скажи, ждать ли эти хотелки и будешь ли делать дополнительное задание?

LeMarck commented:
@bratva, привет!
Да, сделаю))

LeMarck committed:
update

1. Запретить выделение элементов в котле и списке (см user-select)
2. Покрасить каждый элемент в свой стиль (вода - голубая, огонь - красный и тп)

LeMarck committed:
add

3. Оформить котел и полку с элементами

bratva commented:
Работает, но красивее не стало =)
Бутылки съезжают с полок, в котел не влазит, текст не читается.

Ждать еще косметику?

LeMarck committed:
reload

LeMarck committed:
add index.css

LeMarck commented:
@bratva
Поправил кодстайл, чтоб не всё так плохо было))
Я не успеваю доделать всю красоту(((
